docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx

```diff
@@ -78,6 +81,11 @@ The example below shows configuring the StorageManager to upload to the `protect
 </Example>
 
 ### Deprecated props
+
+<Alert variation="warning" heading="Storage Manager is deprecated">
```

Comment:
```suggestion
<Alert variation="warning" heading="Storage Manager renamed">
```

Reply:
let's move this alert to the top of the page

Reply:
Done and done